detect-nuget-inspector/detect-nuget-inspector/Inspection/Util/ExcludedDependencyTypeUtil.cs

```diff
@@ -0,0 +1,17 @@
+namespace Synopsys.Detect.Nuget.Inspector.Inspection.Util
+{
+
+    static class ExcludedDependencyTypeUtil
+    {
+        public static bool isDependencyTypeExcluded(String excludedDependencyTypes)
```

Comment:
Should this be named something like "isDevDependencyTypeExcluded" instead?

Reply:
I have kept this name intentionally as if there are checks for more dependency types added in the future.

Reply:
That makes sense. Should we then change this method take an additional argument for the type we're interested in instead of checking against hard-coded "DEV"?

Reply:
Only one argument is enough for now as we have only one dependency type. The argument which is being checked against hardcoded "DEV" is coming from Detect and passed along this method for being checked so it does not make sense to add one more argument right now.

Reply:
What are the benefits of this future-proofing, then? Unless I'm missing something, the main benefit of future-proofing here would be ensuring that usages of this method do not need to change in case more types are added down the road.

Reply:
I see what you mean, sorry got confused by the other parameter with a different thing, will add that in the next commit.